core/dom/cell.js

```diff
@@ -8,10 +8,12 @@ export class Cell extends Element {
 		this.context = context;
 		this.rowIndex = rowIndex;
 		this.columnIndex = columnIndex;
+		const model = this.model();
+		this.columnKey = model ? model.columnKey : null;
```

Comment:
revert changes u can get column key throught the model

Reply:
fixed